test/vsphere/e2e.yml

```diff
@@ -0,0 +1,125 @@
+name: $PLATFORM_CLUSTER_ID
+domain: $PLATFORM_CLUSTER_ID.lab.flanksource.com
+consul: 10.100.0.13
+hostPrefix: k8s
+datacenter: lab
+ldap:
+  disabled: true
+kubernetes:
+  version: v1.16.4
+versions:
+  gomplate: v3.5.0
+  konfigadm: v0.3.6
+  jb: v0.1.0
+  jsonnet: 0.14
+  sonobuoy: 0.16.4
+  govc: v0.20.0
+  gojsontoyaml: master
+  kubectl: v1.15.3
+  pgo: 4.2.0
+  helm: v2.13.0
+  velero: v1.2.0
+  expenv: v1.2.0
+  ketall: v1.3.0
+podSubnet: 100.200.0.0/16
+serviceSubnet: 100.100.0.0/16
+opa:
+  version: 0.13.5
+  kubeMgmtVersion: 0.8
+calico:
+  ipip: Never
+  vxlan: Never
+  version: v3.8.2
+nodeLocalDNS:
+  disabled: false
+monitoring:
+  version: dfb626837f04756ed5a8805845f51ebd29d342ec
+pgo:
+  version: 4.2.0
+  imageVersion: 4.2.0
+  primaryStorage: local-path
+  xlogStorage: local-path
+  backupStorage: local-path
+  replicaStorage: local-path
+  backrestStorage: local-path
+  storage:
+    xlogstorage:
+      AccessMode: ReadWriteOnce
+      Size: 2G
+      StorageType: dynamic
+      storageClass: local-path
+      Fsgroup: 26
+    local-path:
+      AccessMode: ReadWriteOnce
+      Size: 50G
+      StorageType: dynamic
+      StorageClass: local-path
+      Fsgroup: 26
+    s3:
+      AccessMode: ReadWriteOnce
+      Size: 50G
+      StorageType: dynamic
+      StorageClass: s3
+    nfs:
+      AccessMode: ReadWriteOnce
+      Size: 50G
+      StorageType: dynamic
+      StorageClass: nfs
+harbor:
+  replicas: 1
+  version: v1.10.0
+  chartVersion: v1.3.0
+s3:
+  endpoint: http://minio.minio.svc:9000
+  externalEndpoint: minio.127.0.0.1.nip.io
+  bucket: harbor
+  access_key: minio
+  secret_key: minio123
+  region: us-east1
+velero:
+  bucket: velero
+fluentd-operator:
+  version: v1.11.0
+  repository: jvassev/kube-fluentd-operator
+  elasticsearch:
+    url: log-es-http.eck.svc.cluster.local
+    user: elastic
+    password: elastic
+    port: 9200
+    scheme: https
+eck:
+  version: 1.0.0
+ca:
+  cert: $PLATFORM_CA
```

Comment:
```suggestion
  cert: .certs/ca.crt.pem
```

Reply:
would not be better to pass here as env variable generated inside the script?